test/Atmos/EDMF/closures/mixing_length.jl

```diff
@@ -63,14 +59,8 @@ function mixing_length(
     L_Nˢ = coeff * Nˢ_fact + ml.max_length * (FT(1) - Nˢ_fact)
 
     # compute L2 - law of the wall
-    surf_vals = subdomain_surface_values(
-        m.turbconv.surface,
-        m.turbconv,
-        m,
-        gm,
-        gm_aux,
-        m.turbconv.surface.zLL,
-    )
```

Comment:
I removed this because I don't think we can call `subdomain_surface_values` here: The states passed to that function are assumed to be at the surface, not at any height. (This function was not used though, it was a placeholder for later use).